build-docker-plotman.sh

```diff
@@ -2,22 +2,20 @@
 
 DOCKER_REGISTRY="<docker-registry>"
 PROJECT="chia-plotman"
-TAG="plotter"
+TAG="latest"
 BASE_CONTAINER="ubuntu:20.04"
-CHIA_GIT_REFERENCE="1.1.7"
+CHIA_GIT_REFERENCE="1.2.3"
 
 # The UID/GID should match the 'chia' owner of the directories on the host system
-UID=10001
-GID=10001
-
-docker rmi ${LOCAL_REGISTRY}/${PROJECT}:${TAG}
+bUID=10001
```

Comment:
Not clobbering `UID` sounds good.  Does the `b` refer to anything in particular?

Reply:
Just build_UID essentially. I'll make it more descriptive :-)